week-3/todo-list/script.js

```diff
@@ -1,25 +1,126 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
+document.addEventListener("DOMContentLoaded", function () {
+  const todoForm = document.getElementById("todo-form");
+  const todoInput = document.getElementById("todoInput");
+  const deadlineInput = document.getElementById("deadlineInput");
+  const todoList = document.getElementById("todo-list");
+  const removeAllCompletedButton = document.getElementById(
+    "remove-all-completed"
+  );
+  const addTodoBtn = document.getElementById("addTodoBtn");
 
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
-let todos = [
-  { task: "Wash the dishes", completed: false },
-  { task: "Do the shopping", completed: false },
-];
+  let todos = [
+    // { task: "Wash the dishes", completed: false, deadline: "" },
```

Comment:
PR should not contain any commented out code. If the code is no longer used, just remove it.